tasks-desktops.yml

```diff
@@ -37,8 +37,6 @@
     name: lnls-ans-role-sirius-bbb
 - import_role:
     name: lnls-ans-role-octave
-- import_role:
-    name: lnls-ans-role-epics-mca
```

Comment:
We are we removing this role?

Reply:
yes, we decided not to install any of currently unused epics packages and libraries.

Reply:
But this is an EPICS extension used by octave.It's not a module used to build IOCs

Reply:
reverted back to using it in the playbook...